pkg/buildpack/buildpack.go

```diff
@@ -356,10 +350,6 @@ func ToLayerTar(dest string, module BuildModule) (string, error) {
 }
 
 func ToNLayerTar(dest string, module BuildModule) ([]ModuleTar, error) {
-	if !module.ContainsFlattenedModules() {
-		return handleSingleOrEmptyModule(dest, module)
-	}
```

Comment:
The recursive function can handle the case where the module contains a single buildpack